src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -712,10 +726,21 @@ export default defineComponent({
     },
 
     updateVolume: function (_event) {
-      // 0 means muted
       // https://docs.videojs.com/html5#volume
-      const volume = this.player.muted() ? 0 : this.player.volume()
-      sessionStorage.setItem('volume', volume)
+      if (sessionStorage.getItem('muted') === 'false' && this.player.volume() === 0) {
```

Comment:
Why are you checking the session storage here, instead of the player?

Reply:
sessionStorage is not set to true when we drag the slider to 0 (it does get set to true if we press 'm' or click on mute though) so in those cases I am setting volume to defaultVolume set by user in settings. This should happen when updateVolume function is called by volumechange event I reckon? Because if s2these conditions are checked in player, then the feature doesn't work as intended

Reply:
Video.js doesn't change anything in session storage. the values in session storage are set once at startup by the code in the settings file and then updated by the code in this function every time the player is muted/unmuted or the volume of the player changes. Without this updateVolume function nothing would be changed in session storage after setting the initial values at startup in the settings file.

Reply:
Are there any changes that I should do here? I tried checking sessionStorage in player and update player volume accordingly but it doesn't work as intended.